src/Models/ShallowWaterModels/solution_and_tracer_tendencies.jl

```diff
@@ -4,17 +4,8 @@ using Oceananigans.Operators
 
 @inline squared(i, j, k, grid, ϕ) = ϕ[i, j, k]^2
```

Comment:
```suggestion
@inline squared(i, j, k, grid, ϕ) = @inbounds ϕ[i, j, k]^2
```

Reply:
Happy to make the change, but I admit I don't know why we use @inbounds sometimes and not others.